docs/modeling/model-components/attributes/coalesce.md

```diff
@@ -9,4 +9,15 @@ Some types and members will be implicitly included in generation - for example,
 
 On the other hand, [Methods](/modeling/model-components/methods.md) on these types will not have endpoints generated unless they are explicitly annotated with `[Coalesce]` to avoid accidentally exposing methods that were perhaps not intended to be exposed.
 
+## Type-Level Usage
+
+When used on a class, the `[Coalesce]` attribute will cause the type to be included in Coalesce generation:
+
+- If the type also has `[Service]`, it becomes a [Service](/modeling/model-types/services.md).
+- If the type also has `[StandaloneEntity]`, it becomes a [Standalone Entity](/modeling/model-types/standalone-entities.md).
+- If used on a `DbContext`, it enables the context and its entities for Coalesce generation.
+- Other specialized interface implementations (like `IDataSource<>`, `IBehaviors<>`, `IClassDto<>`) become their respective component types.
+
+For standalone data classes that should be treated as [Simple Models](/modeling/model-types/simple-models.md), use the `[SimpleModel]` attribute instead.
+
```

Comment:
Remove all this documentation. Do not add any documentation to this file. What you did directly contradicts the line immediately following what you added.